graylog2-web-interface/CONTRIBUTING.md

```diff
@@ -68,6 +68,7 @@ you want to use (class or a functional component with react hooks). When you don
   - `__tests__/ComponentA.test.jsx`
   - `__tests__/ComponentA.test.case1.json`
   - `__tests__/ComponentA.text.case1.result.json`
+- New tests should import the Testing-Library's render function from `wrappedTestingLibrary`.
```

Comment:
Why not even change it to:
> ~~New~~ Tests should import the Testing-Library's render function from `wrappedTestingLibrary`.
At some point now one knows what a new or old test is and it will encourage refactoring of existing tests :)

Reply:
Makes sense!